spec/support/omniauth_helpers.rb

```diff
@@ -0,0 +1,17 @@
+def set_omniauth
+  OmniAuth.config.test_mode = true
+
+  OmniAuth.config.mock_auth[:google_oauth2] = OmniAuth::AuthHash.new(
+  {
+    provider: 'google_oauth2',
+    uid: '123545',
+    info: {
+      name: "Tom Brady",
+      email: "TB12@pats.biz"
+     },
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.